src/strings/PowerFxResources.en-US.resx

```diff
@@ -1798,6 +1798,10 @@
     <value>Numeric value is too large.</value>
     <comment>Error Message.</comment>
   </data>
+  <data name="ErrTextFormatTooLarge" xml:space="preserve">
+    <value>Format string size can't be more than {0} characters.</value>
```

Comment:
>can't [](http://example.com/codeflow?start=30&length=5)

We are very inconsistent with our usage of `can't` and `cannot`. We should choose one format and stick with it. Not needed for this PR, though.